src/components/List/index.tsx

```diff
@@ -66,7 +74,7 @@ function List() {
       {
         list.map((item, index) => (
```

Comment:
When you have id should use it for index

Reply:
en la key burro no en el index